<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>buypanel</class>
 <widget class="QWidget" name="buypanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>702</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGraphicsView" name="imageView">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-10</y>
     <width>971</width>
     <height>741</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Yekan Bakh</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/images/background);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>30</y>
     <width>301</width>
     <height>81</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>301</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>301</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Yekan Bakh</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;</string>
   </property>
   <property name="text">
    <string>فاکتور خرید</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>110</y>
     <width>751</width>
     <height>20</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="productLabel">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>160</y>
     <width>351</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Yekan Bakh</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;</string>
   </property>
   <property name="text">
    <string>صابون گلنار</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QSpinBox" name="productCount">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>160</y>
     <width>100</width>
     <height>70</height>
    </rect>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
  </widget>
  <widget class="QLabel" name="productPrice">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>160</y>
     <width>161</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Yekan Bakh</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;</string>
   </property>
   <property name="text">
    <string>15000 تومان</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="productLabel_3">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>170</y>
     <width>101</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Yekan Bakh</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;</string>
   </property>
   <property name="text">
    <string>X</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>250</y>
     <width>751</width>
     <height>20</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="productLabel_4">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>300</y>
     <width>211</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Yekan Bakh</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;</string>
   </property>
   <property name="text">
    <string>جمع کل خرید</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="total">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>300</y>
     <width>391</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Yekan Bakh</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;</string>
   </property>
   <property name="text">
    <string>30000 تومان</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="buyBtn">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>390</y>
     <width>751</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Yekan Bakh</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:&quot;green&quot;;
color:&quot;white&quot;;</string>
   </property>
   <property name="text">
    <string>پرداخت</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
